scanner/src/lib/dataproducts/battery/CarbonFootprint_v0_1.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
```

Comment:
This file is clearly failing to follow the obvious previous naming standard for them

![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/174a155a-4d18-44bb-97bc-bde9554b468d)